src/shared/assets.ts

```diff
@@ -344,6 +344,33 @@ export class AssetGenerator {
 
         return result;
     }
+
+    public getAssetsPathAndProgram(): [string, string] {
+        let assetsPath = ``;
+        this.executableProjects.forEach((project) => {
+            if (project.isWebAssemblyProject) {
+                assetsPath += path.join(path.dirname(project.outputPath), path.sep);
+                assetsPath += ';';
+            }
+        });
+        assetsPath = assetsPath.slice(0, -1);
+        return [assetsPath, this.executableProjects[0].outputPath];
+    }
+
+    public isDotNet9OrNewer(): boolean {
+        let ret = false;
+        this.executableProjects.forEach((project) => {
+            if (project.isWebAssemblyProject) {
+                const projectFileText = fs.readFileSync(project.projectPath, 'utf8');
+                ret =
+                    projectFileText.toLowerCase().indexOf('net8.0') < 0 &&
```

Comment:
Similar comment here about not just looking for the names of various frameworks that we don't support but instead looking for ones we do support.

Reply:
I was trying to avoid need to change it everytime that we ship a new product? like when we ship .net 10 we will need to add it here?

Reply:
Ah okay, ignore me, a regex will avoid it. > 9 should be valid.